packages/semver/src/builders/version/builder.ts

```diff
@@ -63,7 +66,8 @@ export function runBuilder(
 
       return concat(
         runStandardVersion$,
-        ...(push && dryRun === false ? [pushToGitRemote$] : [])
+        ...(push && dryRun === false ? [pushToGitRemote$] : []),
+        pluginHandler.publish() // @todo handle dryRun
```

Comment:
Shouldn't we `defer(() => ...publish())`? 
We might need a test that makes sure that `publish` is only called after `version` finishes and succeeds.

Reply:
The publish function already defer it result. I agree, I will add a test for this.